services/meta/renameFile_unix.go

```diff
@@ -0,0 +1,12 @@
+// +build !windows
+
+package meta
+
+import (
+	"os"
+)
+
+// rename the source to target using os function
```

Comment:
The first string of the docstring comment should match the name of the function.